### PR TITLE
Feature: create container feature(UI) in group single page

### PR DESCRIPTION
**Pull request by ShoeheyOt**
## Overview

implement displaying a drawer for creating container when plus icon touched, only UI part.

## Changes

-

## Review points

-

## Screen Captures
[createContainerDrawerUI.webm](https://github.com/nishiki-tech/nishiki-frontend/assets/142355969/b428978b-7721-4a29-847f-51c10fe7b6e4)

## References

-

## Notes

-

## Assignee Checklist:

- [X] The naming convention of the PR title is correct (See the comment at the top of this template)
- [X] The base branch is correct (See: [Types of Branches](https://github.com/nishiki-tech/nishiki-frontend/blob/develop/docs/CONTRIBUTING.md#types-of-branches))
- [ ] The branch name follows the [Branch Naming Conventions](https://github.com/nishiki-tech/nishiki-frontend/blob/develop/docs/CONTRIBUTING.md#branch-naming-conventions)
- [x] The correct assignees and reviewers have been designated for this PR
- [X] The coding style follows the [Coding Style Guide](https://github.com/nishiki-tech/nishiki-frontend/blob/develop/docs/STYLEGUIDE.md)
- [x] All the related issues are associated with this PR
- [ ] All criteria in the associated issues are met (please tick the checkboxes)
- [X] My changes do not generate new warnings or errors (especially in the console)

## Reviewer Checklist:

- [ ] Double-check the "Assignee Checklist"
- [x] The code follows the generic best practices and our [Coding Style Guide](https://github.com/nishiki-tech/nishiki-frontend/blob/develop/docs/STYLEGUIDE.md)
- [ ] The code is well-commented and easy to understand
- [x] The UI changes accurately reflect the provided design

### CODE REVIEW

File: src/features/groups/components/ContainerList/CreateContainerButton/CreateContainerBottomDrawerContent.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+import { zodResolver } from '@hookform/resolvers/zod';
+import { FC, useEffect } from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { z } from 'zod';
+
+interface ICreateContainerDrawerContentProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
```
Comment (nick-y-ito) [inline src/features/groups/components/ContainerList/CreateContainerButton/CreateContainerBottomDrawerContent.tsx:29]: Could you please also add TSDoc to each property of this interface? There are many existing interfaces that still do not have TSDoc, though. It's a best practice to always add TSDocs to interfaces, as well as functions.

You can get an idea by visiting the `src/components/layouts/MobileLayout.tsx` file.
Also this PR (https://github.com/nishiki-tech/nishiki-frontend/pull/246/files) includes a lot of TSDocs for interfaces.

**Review by nick-y-ito — CHANGES_REQUESTED** [review]
@ShoeheyOt 
Thank you for your implementation! Although I left many comments, they are all acceptable, and the appearance and the behavior look very good. Great job!! Please kindly take a look at the comments.

However, I would like you to give the "Assignee's Checklist" your full attention. Three of them do not meet the conditions.

- [ ] The branch name follows the [Branch Naming Conventions](https://github.com/nishiki-tech/nishiki-frontend/blob/develop/docs/CONTRIBUTING.md#branch-naming-conventions)
- [ ] All the related issues are associated with this PR
- [ ] All criteria in the associated issues are met (please tick the checkboxes)

File: src/features/groups/components/ContainerList/CreateContainerButton/CreateContainerBottomDrawerContent.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+              render={({ field }) => (
+                <FormItem>
+                  <FormLabel required>Container name</FormLabel>
+                  <FormControl>
+                    <Input {...field} />
+                  </FormControl>
+                  <FormMessage />
+                </FormItem>
+              )}
+            ></FormField>
```
[the remark this answers is not in the document]
Comment (ShoeheyOt) [reply src/features/groups/components/ContainerList/CreateContainerButton/CreateContainerBottomDrawerContent.tsx:81]: @nick-y-ito Thank you for reviewing!
I have changed it to self-closing, can I change it for other file too? same things happens in below file
src/features/groups/components/CreateGroupButton/CreateGroupDrawerContent.tsx

File: src/features/groups/components/ContainerList/CreateContainerButton/CreateContainerBottomDrawerContent.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+interface ICreateContainerDrawerContentProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+export const CreateContainerDrawerContent: FC<ICreateContainerDrawerContentProps> = ({
+  isOpen,
+  onClose,
+}) => {
```
[the remark this answers is not in the document]
Comment (ShoeheyOt) [reply src/features/groups/components/ContainerList/CreateContainerButton/CreateContainerBottomDrawerContent.tsx:34]: @nick-y-ito Thanks you for reviewing!
can I change below file as well? src/features/groups/components/CreateGroupButton/CreateGroupDrawerContent.tsx

File: src/features/groups/components/ContainerList/CreateContainerButton/CreateContainerBottomDrawerContent.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (nick-y-ito) [reply src/features/groups/components/ContainerList/CreateContainerButton/CreateContainerBottomDrawerContent.tsx]: Sure! We would rather appreciate it 😁 
Please leave a comment if you make a change that is not related to the primary task.

File: src/features/groups/components/ContainerList/CreateContainerButton/CreateContainerBottomDrawerContent.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (nick-y-ito) [reply src/features/groups/components/ContainerList/CreateContainerButton/CreateContainerBottomDrawerContent.tsx]: Sure! We would rather appreciate it 😁 
Please leave a comment if you make a change that is not related to the primary task.

**Review by nick-y-ito — APPROVED** [review]
@ShoeheyOt 
Thank you for fixing all the points I commented on earlier!
Now, this PR looks good to go 😄

**Pull request merged**